src/history/hs.c

```diff
@@ -1414,7 +1414,7 @@ __wt_history_store_verify_one(WT_SESSION_IMPL *session)
      * in the history store.
      */
     memset(&hs_key, 0, sizeof(hs_key));
-    cursor->set_key(cursor, btree_id, &hs_key, 0, 0, 0, 0);
```

Comment:
Is this a straight-up bug in the current code?

Reply:
Yes -- no bad effects though, just a typo.